ras/smtstate.py

```diff
@@ -67,3 +73,26 @@ def test(self):
                     self.log.info("SMT load is successful for SMT=%s" % j)
                 else:
                     self.fail("smt load failed")
+
+    def test_smt_state_switching(self):
+        """
+        Test to check the time taken for switching between different
+        SMT levels. The time taken must not exceed beyond 5 minutes.
```

Comment:
not sure if this statement holds good and may be reworded like 

SMT levels. The time taken in general cases should not exceed 5 minutes.

Reply:
Noted.

Reply:
Done